package.json

```diff
@@ -32,11 +32,13 @@
     "prettier": "prettier --write 'src/**/*.ts'; prettier --write 'src/test/**/*.ts'",
     "prepack": "yarn lint && tsc && cp -R src/helpers/ lib/helpers && cp -R src/scripts/ lib/scripts",
     "setChromeDriverVersion": "node src/scripts/SetDriverVersionToDotEnv.js",
-    "test": "DEFAULT_TIME_OUT=3000 yarn setChromeDriverVersion; wdio src/test/wdio.conf.js --suite regression",
+    "test": "DEFAULT_TIME_OUT=3000 yarn setChromeDriverVersion; cleanSeleniumPort; wdio src/test/wdio.conf.js --suite regression",
```

Comment:
shouldn't it be with yarn? like that....
`yarn cleanSeleniumPort `

Reply:
fixed